kalite/distributed/settings.py

```diff
@@ -98,7 +98,7 @@ def USER_FACING_PORT():
 CENTRAL_SERVER_HOST   = getattr(local_settings, "CENTRAL_SERVER_HOST",   ("adhoc.%s:7007" if DEBUG else "kalite.%s") % CENTRAL_SERVER_DOMAIN)
 CENTRAL_WIKI_URL      = getattr(local_settings, "CENTRAL_WIKI_URL",      "http://kalitewiki.%s/" % CENTRAL_SERVER_DOMAIN)
 
-KHAN_EXERCISES_DIRPATH = os.path.join(os.path.dirname(__file__), "static", "khan-exercises")
+KHAN_EXERCISES_DIRPATH = os.path.join(os.path.dirname(__file__), "static", "perseus", "ke")
```

Comment:
Actually, we'll wanna move this definition out of here and into kalite/settings.py.